meshtastic/module_config.proto

```diff
@@ -201,6 +201,15 @@ message ModuleConfig {
      * Only applicable if the board uses pull-up resistors on the pin
      */
     bool use_pullup = 8;
+
+    /* If this is set, message are send to channel set in "sendTo" else a "NodeID"*/
+    bool sendTochannel = 9;
```

Comment:
Don't mind changing it but it won't send on Primary channel (index 0) to prevent spamming.
`if (moduleConfig.detection_sensor.sendTo != 0x00)`

Reply:
Why would that limitation be there? Someone's primary channel may be a private channel.

Reply:
It looks like in your PR you're already checking for `!isDefaultChannel()`.

Reply:
You are right, was using that one to check you it was set when it was char.

Do have another check `if (!channels.isDefaultChannel(msgDestination))` to prevent spamming default public channel.